pycti/api/opencti_api_client.py

```
@@ -249,6 +266,167 @@ def __init__(
+        :type https_ca_certificates: str
+        :return: Certificate content in PEM format or None if invalid
+        :rtype: str or None
+        """
+        # Strip whitespace once at the beginning
+        stripped_https_ca_certificates = https_ca_certificates.strip()
+        
+        # Check if it's inline certificate content (starts with PEM header)
+        if stripped_https_ca_certificates.startswith("-----BEGIN CERTIFICATE-----"):
+            self.app_logger.debug(
+                "HTTPS_CA_CERTIFICATES contains inline certificate content"
+            )
+            return https_ca_certificates
+
+        # Check if it's a file path
+        if os.path.isfile(stripped_https_ca_certificates):
+            cert_file_path = stripped_https_ca_certificates
+            try:
+                with open(cert_file_path, "r") as f:
+                    cert_content = f.read()
+                    # Validate it's actually a certificate
+                    if "-----BEGIN CERTIFICATE-----" in cert_content:
+                        self.app_logger.debug(
+                            "HTTPS_CA_CERTIFICATES contains valid certificate file path",
+                            {"file_path": cert_file_path},
+                        )
+                        return cert_content
+                    else:
+                        self.app_logger.warning(
+                            "File at HTTPS_CA_CERTIFICATES path does not contain valid certificate",
+                            {"file_path": cert_file_path},
+                        )
+                        return None
+            except Exception as e:
+                self.app_logger.warning(
+                    "Failed to read certificate file",
+                    {"file_path": cert_file_path, "error": str(e)},
+                )
+                return None
+
+        # Neither inline content nor valid file path
+        return None
```

I think it'd be better to throw an error here instead of returning None. When calling _get_certificate_content, we're sure that https_ca_certificates is currently set, so we're waiting for a valid certificate to come out of it. If we can't find a valid certificate from it, it means that this was an unexpected